todo_app/app.py

```diff
@@ -8,4 +12,38 @@
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items = get_items()
+
+    items = sorted(items, key=lambda items : items['status'], reverse=True)
+
+    return render_template('index.html', ToDos = items)
+
+@app.route('/submit', methods=['POST','GET'])
+def submit():
+    add_item(request.form.get('title'))
+    return redirect('/')
+
+@app.route('/todo/<id>')
+def view_item(id):
+    item = get_item(id)
+    not_started = False
+    started = False
+    done = False
+
+    if item['status'] == "Not Started":
+        not_started = True
+    if item['status'] == "Started":
+        started = True
+    if item['status'] == "Done":
+        done = True     
+    return render_template('item.html', ToDo = get_item(id), not_started  = not_started, started = started, done = done)
+
+@app.route('/submit_item/<id>', methods=['POST', 'GET'])
```

Comment:
Your two POST routes could probably have clearer paths. Not super obvious what `/submit` and `/submit_item` do just by looking at the paths.

Could have been e.g. `/new_item` and `/update_item`